### ISSUE TITLE
[Enhancement] Refactor unit tests for typography in general

### ISSUE THREAD

**Issue by jkaltoft** [kirbydesign/designsystem#3580]
- [x] Rethink how typography should be tested when font sizes are not absolute
- [x] Rewrite unit tests

-----

This issue is based on the findings of research https://github.com/kirbydesign/designsystem/issues/3568

Read more:
- [Success Criterion 1.4.4 Resize Text](https://www.w3.org/TR/WCAG22/#resize-text)
- [Understanding SC 1.4.4](https://www.w3.org/WAI/WCAG22/Understanding/resize-text.html)
- About lenght units: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/length) and [W3C](https://www.w3.org/TR/css-values-4/#lengths)
